src/nuget/Microsoft.DotNet.BuildTools.TestSuite.nuspec

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.TestSuite</id>
```

Comment:
It might be good to update the test-runtime here to serve as a sample, what do you think @weshaggard?
Can you double check there is no overlap with dlls in TestSuite.  Also, if this needs to depend on TestSuite please make it do so.